decompress_tar.go

```diff
@@ -90,8 +91,16 @@ func untar(input io.Reader, dst, src string, dir bool) error {
 			return err
 		}
 
-		// Set the access and modification time
-		if err := os.Chtimes(path, hdr.AccessTime, hdr.ModTime); err != nil {
+		// Set the access and modification time if valid, otherwise default to current time
+		aTime := time.Now()
```

Comment:
Capture now outside the for loop (https://github.com/hashicorp/go-getter/pull/106/files#diff-0d8d3e8dc13e64092481d7dd63191a0fR17) and use it here and at https://github.com/hashicorp/go-getter/pull/106/files#diff-0d8d3e8dc13e64092481d7dd63191a0fR112.